patientIdentifierType.md

```diff
@@ -0,0 +1,44 @@
+# PatientIdentifierType
+
+## Overview
+Administrators define what types of identifiers they will collect. These range from National ID numbers, to driver's license numbers, to per-hospital medical record numbers.
+
+### Properties on PatientIdentifierType:
+
+* format: a regular expression defining what the identifier text should contain.
+
+* formatDescription: the text an admin can enter describing the regex format they just added
+
+* required: a true/false whether every patient MUST have this type
+
+* scheckDigit: a true/false whether this identifier has a checkdigit at the end.
+
+* validator: full class name of the IdentifierValidators
+
+* locationBehavior: REQUIRED=there must be patient_identifier.location_id defined by the user for this identifier or NOT_USED = patient_identifier.location_id is null
```

Comment:
"REQUIRED" if a location must be associated with the identifier; "NOT_USED" if the identifier does require a location.